vue/components/ui/molecules/select/select.vue

```diff
@@ -373,6 +357,7 @@ export const Select = {
             }
             this.keyBuffer += key.toUpperCase();
             this.keyTimestamp = Date.now();
+            this.$emit("select:keydown", this.keyBuffer);

```

Comment:
`this.$emit("select:keydown", key, this.keyBuffer);`

You should also emit the event and the key, As this `onKey` handler doesn't receive the `event` we are only emitting `key` :/